src/Discord.Net/Entities/Guilds/IGuild.cs

```diff
@@ -0,0 +1,90 @@
+// todo: impl, docs
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Wumpus;
+using Wumpus.Entities;
+
+namespace Discord
+{
+    public interface IGuild : ISnowflakeEntity, IDeletable
+    {
+        // todo: IAudioClient
+        string Name { get; }
+        ulong? ApplicationId { get; }
+        ulong EveryoneRoleId { get; }
+        ulong OwnerId { get; }
+
+        int AFKTimeout { get; }
+        bool IsEmbeddable { get; } //review: is this still applicable
+        string VoiceRegionId { get; }
+
+        DefaultMessageNotifications DefaultMessageNotifications { get; }
+        MfaLevel MfaLevel { get; }
+        VerificationLevel VerificationLevel { get; }
+        ExplicitContentFilter ExplicitContentFilter { get; }
+
+        string? IconId { get; }
+        string? IconUrl { get; }
+        string? SplashId { get; }
+        string? SplashUrl { get; }
+
+        bool Attached { get; } // review: how to make this more consumable
+
+        ulong? AFKChannelId { get; }
+        ulong? DefaultChannelId { get; } // todo: can we expose this synchronously? needs access to channels
```

Comment:
Since this has been scrapped from the client should probably just leave this up to the user to implement/have an extension method for it.